.circleci/config.yml

```diff
@@ -251,6 +251,16 @@ commands:
 
             ./scripts/build-release.sh filcrypto $(cat ./rust-toolchain) build --verbose --locked --no-default-features --features multicore-sdr,opencl
             ./scripts/package-release.sh $TARBALL_PATH
+      - run:
+          name: Build the optimized release (cuda)
```

Comment:
do we not run tests on these?

Reply:
Apparently not, only default features seem to have the tests run

Reply:
I'll remove those builds and swap for tests

Reply:
Failed with no nvcc, as I originally expected. Can try to add cuda via apt I guess

Reply:
No nvidia-cuda-toolkit.  What am I missing?

Reply:
Ah, I see, the ubuntu image

Reply:
No go yet, I've removed everything except for the blstrs update for now.  I'll create an issue for CI testing of CUDA but don't want it to block the release

Reply:
On CI you need to make sure that the CUDA stuff (`/usr/local/cuda-11.2/bin`) is in the `PATH`, see e.g. https://github.com/filecoin-project/rust-gpu-tools/blob/d3b36477c7d2017e2b7ea284fd91342a667835db/.circleci/config.yml#L26